src/interfaces/slack_bot/src/constants.ts

```diff
@@ -0,0 +1,151 @@
+import { Action } from './utils/actions';
+
+export const COMMAND_SUFFIX = process.env.COMMAND_SUFFIX || '';
+export const COHERE_API_TIMEOUT_SECONDS = 60 * 5; // 5 minutes
+
+// todo: change to follow the format of the other envs above
+export const DEFAULT_CHAT_TEMPERATURE = 0.3;
```

Comment:
Can we not pass in a temp?

Reply:
The temperature can be changed for a channel with a `/set-temperature` command. This is the default value.

<img width="732" alt="Screenshot 2024-06-13 at 10 51 37 AM" src="https://github.com/cohere-ai/cohere-toolkit/assets/1205575/e188a2bb-25e6-4022-a0de-c608b97b2234">